R/create.ccf.summary.heatmap.R

```diff
@@ -45,7 +45,7 @@ create.ccf.summary.heatmap <- function(
     names(sample.df)[2] <- names(clone.df)[2] <- 'nsnv';
 
     if (!is.null(clone.order) & !is.null(sample.order)) {
-        arr                 <- arr[clone.order, sample.order];
+        arr                 <- arr[clone.order, rev(sample.order)];
```

Comment:
I assume this is because the heatmap goes bottom-to-top>

Reply:
yepp